README.md

```diff
@@ -16,7 +16,10 @@ Note: `version.scala.binary` refers to major releases of scala ie. 2.11, 2.12 or
 ```xml
 <plugin>
     <groupId>org.antipathy</groupId>
-    <artifactId>mvn-scalafmt_${version.scala.binary}</artifactId>
+    <!-- Always use mvn-scalafmt_2.13.
```

Comment:
Could you change this to 'mvn-scalafmt_2.13 is fine for almost all use cases'.  'Always' sounds a little aggressive to me.